src/Microsoft.DncEng.SecretManager/SecretTypes/AzureDevOpsAccessToken.cs

```diff
@@ -28,13 +40,46 @@ public AzureDevOpsAccessToken(ISystemClock clock, IConsole console)
         Console = console;
     }
 
-    protected override async Task<SecretData> RotateValue(Parameters parameters, RotationContext context, CancellationToken cancellationToken)
+    private static class Constants
```

Comment:
Does this really need to be a class? :-)

Reply:
No, not really. I'll move then up one nesting level.